.circleci/config.yml

```diff
@@ -0,0 +1,121 @@
+version: 2.1
+orbs:
+  node: circleci/node@4.1.0
```

Comment:
the circleci images are being deprecated, you might want to use cimg instead

https://circleci.com/developer/images/image/cimg/node

PS: we need to update config files in the other repos as well

Reply:
I tried to use the new cimg docker image but was getting errors with node-sass so I reverted back to the old circleci node docker image 😞

Reply:
good to know, with the same version of the image ?

Reply:
With `cimg/node:14.15` since it doesn't seem like the `latest` tag exists for the new docker images.